src/components/navigation/components/LoginController.js

```diff
@@ -86,13 +86,26 @@ export class LoginController extends React.PureComponent {
 
     componentDidMount () {
         const { auth0ClientID, auth0Domain, auth0Options } = this.props;
-        require.ensure(["auth0-lock"], (require) => {
+
+        import(
```

Comment:
Using more up-to-date syntax for [code-splitting / dynamic importing](https://webpack.js.org/guides/code-splitting/#dynamic-imports). Still need to figure out how to best grab multiple dependencies/libraries at once (for HiGlass in 4DN portal).